auth/auth/app/schemas/user.py

```diff
@@ -4,6 +4,7 @@
 from beanie import Indexed
 from common.configs.mongo_document import MongoDocument
 from common.enums.plan import Plans
+from common.enums.roles import Roles
```

Comment:
Remove this unused import

Reply:
Removed the unused import and fixed google oauth signup proxy error.